css/files_lock.css

```diff
@@ -4,7 +4,7 @@ tr[data-locked='true'] .fileactions .action.action-menu.permanent {
 }
 .locking-inline-state.icon-password {
 	padding: 17px 23px;
-	margin-right: -45px;
+	margin-right: -49px;
```

Comment:
Unfortunately this has a similar issue with entries that are not shared:

![Screenshot 2023-11-06 at 09 16 17](https://github.com/nextcloud/files_lock/assets/3404133/d63ef8b0-2d52-40d0-8685-8239ab1f1e4a)

The following style changes seem to work though:
![Screenshot 2023-11-06 at 09 15 52](https://github.com/nextcloud/files_lock/assets/3404133/bcaf10a5-5db6-47e1-8ff9-f924f353f360)

Reply:
Check last commit. Both UI content (shared and not shared) should display lock properly, including in grid view.